content/swift/concepts/sets/terms/remove.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.remove()'
+Description: 'Removes an element at a specified index.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.remove()`** method removes an element at a specified index.
+
+## Syntax
+
+```pseudo
+setName.remove(at: index)
+```
+
+## Example
+
```

Comment:
```suggestion

In the example below, The `"autumn"` element in the `weather` array is removed:

```

Reply:
Thank you I fixed this !

Reply:
Hmmm it looks like the text is still not moved up here?

Reply:
I thought you pressed it. I'm sorry. I just posted it Please check

Reply:
Hey @rriver2 it looks like you linked to the official docs for the `.remove()` method for _arrays_ but the issue you were assigned to pertains to the `.remove()` method for _sets_.

See here: https://developer.apple.com/documentation/swift/set/remove(_:)-8p2tv

Reply:
Based on this, we could call it either `value` or `element` but I think `index` makes more sense for an entry pertaining to arrays, not sets. (Codecademy Docs already seems to have [an existing `.remove()` array method entry](https://www.codecademy.com/resources/docs/swift/arrays/remove))